graph/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "sapphire-proxy",
+  "version": "1.0.0",
```

Comment:
Do me a solid and add `"type": "module"` and then use ESM in `index.js`?

Reply:
Maybe I am missing something but isn't sapphire client lib using CommonJS?

Reply:
> It [provides both](https://github.com/oasisprotocol/sapphire-paratime/blob/main/clients/js/package.json#L25-L31). Older version of Node don't support ESM, but the latest LTS does, and since ESM is the rapidly advancing standard, we should prefer to use it.

Well, I am not really experienced with ESM outside of TypeScript but I am getting following error when importing oasis client lib:
<img width="489" alt="image" src="https://user-images.githubusercontent.com/104679740/192525590-7818201c-209d-44ae-9f32-2df2820e07be.png">

But if I remove import of Oasis lib and keep other imports - it works:
<img width="322" alt="image" src="https://user-images.githubusercontent.com/104679740/192525902-a9daf296-c936-4fc4-9c5e-438f0b6512fd.png">

So I am a bit confused with this tbh 😅

Reply:
Ugh, node doesn't like the `js` because `type: module` isn't set in the `sapphire-paratime` package. I know that if I do that, hardhat will start complaining. Typescript doesn't want to mess with file extensions, so I guess I have to do it myself? Give me a little bit and I'll get your PR wrapped up :)

Reply:
> Ugh, node doesn't like the `js` because `type: module` isn't set in the `sapphire-paratime` package. I know that if I do that, hardhat will start complaining. Typescript doesn't want to mess with file extensions, so I guess I have to do it myself? Give me a little bit and I'll get your PR wrapped up :)

So can I resolve this conversation about ESM?

Reply:
Keep it open for a bit please. If you notice #57, I'll be able to add ESM to your PR and resolve this comment myself.